recipes/setuptools/setup.patch

```diff
@@ -0,0 +1,17 @@
+diff --git setup.py setup.py
+index 4ee0ac3..6617197 100755
+--- setup.py
++++ setup.py
+@@ -145,12 +145,8 @@ setup_params = dict(
+         Topic :: Utilities
+         """).strip().splitlines(),
+     extras_require={
+-        "ssl:sys_platform=='win32'": "wincertstore==0.2",
+-        "certs": "certifi==2016.2.28",
+     },
+     dependency_links=[
+-        'https://pypi.python.org/packages/source/c/certifi/certifi-2016.2.28.tar.gz#md5=5d672aa766e1f773c75cfeccd02d3650',
+-        'https://pypi.python.org/packages/source/w/wincertstore/wincertstore-0.2.zip#md5=ae728f2f007185648d0c7a8679b361e2',
```

Comment:
These were added in PR ( https://github.com/conda-forge/setuptools-feedstock/pull/1 ) and the patch was dropped in PR ( https://github.com/conda-forge/setuptools-feedstock/pull/7 ).